second_sql_assignment.sql

```diff
@@ -0,0 +1,232 @@
+CREATE DATABASE TASK2; 
+
+USE TASK2;
+
+CREATE TABLE Employees(
+EmployeeID INT PRIMARY KEY,
+FirstName VARCHAR(50),
+LastName VARCHAR(50),
+Department VARCHAR(50),
+Salary DECIMAL(10,2),
+HireDate Date);
+
+INSERT INTO  Employees VALUES
```

Comment:
always prefer to use the column list while inserting data into a table.

Reply:
Okay